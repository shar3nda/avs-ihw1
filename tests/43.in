10
8 6 0 10 -1 -1 6 9 3 -1