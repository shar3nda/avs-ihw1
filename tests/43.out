0 10 -1 -1 6 9 3 -1